bot/internal/bot/backport.go

```diff
@@ -117,7 +131,7 @@ func (b *Bot) Backport(ctx context.Context) error {
 				RawQuery: url.Values{
 					"expand": []string{"1"},
 					"title":  []string{fmt.Sprintf("[%v] %v", strings.Trim(base, "branch/"), pull.UnsafeTitle)},
-					"body":   []string{fmt.Sprintf("Backport #%v to %v", b.c.Environment.Number, base)},
+					"body":   []string{bodyText},
```

Comment:
You can just add "labels" here than modifying labels.go, would be a little bit cleaner. I actually had this logic implemented and sitting in https://github.com/gravitational/shared-workflows/pull/148 for a while but never got a chance to wrap it up.

Reply:
https://github.com/gravitational/shared-workflows/pull/148/files#diff-6a92e12adffd18e18a79d45839beff2d7b47b7d29ae8335552c7faa19b83966bR127

Reply:
Changed. I had to force push to overwrite the commit, because during writing tests for backport flow it ran real git commands on my machine and overwrote my git user (it's safe now, dry run in tests).